### ISSUE TITLE
gallery_div_id always set to 1

### ISSUE THREAD

**Issue by axel-rank** [variar/grav-plugin-unitegallery#2]
Can't change gallery_ div_id. It's always set to 1.
Should be unite-gallery?

**Comment by variar** [comment]
It seems to work if I put this code in `config/plugins/unitegallery.yaml`
```
enabled: true
gallery_theme: tiles
gallery_div_id: test-id
thumb_width: 600
thumb_height: 600
```

Could you please show some code that you use to change id?

**Comment by axel-rank** [comment]
If I only edit config/plugins/unitegallery.yaml manually it works.
But if I edit in grav admin panel it doesn't. After clicking save id is always set to 1.
In unitegallery.yaml 
gallery_div_id changes to **true**.

**Comment by variar** [comment]
I don't use admin panel. Thanks for reporting this issue. Should be fixed in v1.0.1.

**Comment by axel-rank** [comment]
Yes. Working now. Thanks.
